force-app/main/default/objects/Address__c/fields/Address_Type__c.field-meta.xml

```diff
@@ -12,7 +12,7 @@
             <sorted>false</sorted>
             <value>
                 <fullName>Home</fullName>
-                <default>true</default>
+                <default>false</default>
```

Comment:
Is there a reason for this change?

Reply:
Fixing a bug related to this workflow: https://github.com/SalesforceFoundation/NPSP/pull/6632/commits/4eb9f1acda44638477e5d28c1c92eae79a411ad0

Reply:
Does this change introduce a legacy behavior regression though? I'm pretty sure this was always the original default value. If we change it so there is no longer a default value, doesn't that impact what customers expect to happen?